contentctl/objects/unit_test.py

```diff
@@ -17,19 +16,21 @@ class UnitTest(BaseTest):
     # contentType: SecurityContentType = SecurityContentType.unit_tests
 
     # The test type (unit)
-    test_type: TestType = Field(TestType.UNIT)
+    test_type: TestType = Field(default=TestType.UNIT)
 
+    # TODO (cmcginley): looks like pass_condition has gone the way of the dodo; can I remove?
     # The condition to check if the search was successful
-    pass_condition: Union[str, None] = None
+    pass_condition: str | None = None
```

Comment:
Looks like pass_condition has gone the way of the dodo; can I remove?

Reply:
Pass condition no longer used when building the search: 
https://github.com/splunk/contentctl/pull/260/files#diff-701b2a9b8fd8a8d35960dcf412eb3a87f0639121004f98cb277832179df81faeL1019-L1024

and no longer a field in UnitTest:
https://github.com/splunk/contentctl/pull/260/files#diff-f5f25273cc9fbc9cdc9a96d6c30ed166bd825d73644c366226b94b130e6e7b67L21-L23